src/common/modules/Logger.js

```diff
@@ -13,6 +13,36 @@ const MESSAGE_LEVEL_NAME = Object.freeze({
     [MESSAGE_LEVEL.SUCCESS]: "SUCCESS"
 });
 
+/**
+ * Checks if value is a object.
+ *
+ * @function
+ * @param  {*} value
+ * @returns {boolean}
+ */
+function isObject(value){
```

Comment:
Lodash is partly included in this project. (and `isObject` is already there) You do not need to copy-paste the code here, just use one good `import` statement. (It's used in other modules here, too.)